.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: ci
```

Comment:
@wendigo how much (or how little) of this file needs to exist in master branch for the GHA run to kick off?

Reply:
AFAIR there need to be workflow already committed to the master/main branch so that GHA will pick up one from branch with changes

Reply:
thanks @wendigo 
@tedyu can you please extract addition of this file to a new PR, so we can merge it quickly?
without that GHA won't run.